android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -514,20 +515,18 @@ public boolean startUrlRequest(String url, final long callbackPtr) throws Except
         }
         httpHandler.onRequest(url, new Callback() {
             @Override
-            public void onFailure(Request request, IOException e) {
+            public void onFailure(Call call, IOException e) {
                 onUrlFailure(callbackPtr);
                 //e.printStackTrace();
             }
 
             @Override
-            public void onResponse(Response response) throws IOException {
+            public void onResponse(Call call, Response response) throws IOException {
                 if (!response.isSuccessful()) {
                     onUrlFailure(callbackPtr);
                     throw new IOException("Unexpected response code: " + response);
                 }
-                BufferedSource source = response.body().source();
-                byte[] bytes = source.readByteArray();
-                onUrlSuccess(bytes, callbackPtr);
+                onUrlSuccess(response.body().bytes(), callbackPtr);
```

Comment:
Nice! 

Don't know why I initially did not use `ResponseBody::bytes()`, instead of going to `okio.BufferedSource` and grabbing the bytes from there. Maybe this is also in their new API :D.